pinot-clients/pom.xml

```diff
@@ -48,6 +48,10 @@
       <artifactId>pinot-spi</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+    </dependency>
```

Comment:
Is this required for pinot-java-client too? If not, it would be better to add it to the pinot-jdbc-driver module's pom file.

Reply:
@KKcorps was saying on Slack that it should actually be included via `pinot-spi` but seems to not be anymore

Reply:
`commons-lang3` is already included in the `pinot-spi`. I think the issue is that `pinot-spi` is added as `provided`. @xiangfu0 Any idea why we add `pinot-spi` as `provided` here?